### PR TITLE
isort everything + 1 change

### LINKED ISSUE #75: Couple of bugs in kubernetes.py
- isintance instead of isinstance
- had not imported warnings

These were addressed in #71 which we are closing now so that we can first merge a purely black-only PR, and then we will again file a PR from the `autofmt` branch that specifically addresses these bugs and is easier to review

### PR DESCRIPTION
**Pull request by bolliger32**
OK @delgadom i knowwww you don't like to mix formatter changes and other changes, but #65 actually fixed one of the two bugs I had noticed. The only other one was that warnings was not imported in kubernetes.py but was potentially being called. So this PR contains that one change, and other than that I just called `isort --profile black .` and added all those changes (isort v5 now has profiles so you can make sure that isort doesn't do anything that black would then revert). Importantly it doesn't mess up the importing AFTER sys.path in `conf.py`.

Fixes #75

### CODE REVIEW

File: rhg_compute_tools/kubernetes.py [hunk 1]
```diff
@@ -108,7 +110,7 @@ def get_cluster(
     deploy_mode : str, optional
         Where to deploy the scheduler (on the same pod or a different pod)
     idle_timeout : str, optional
-        Number of seconds without active ommunication with the client before the 
+        Number of seconds without active ommunication with the client before the
```
Comment (delgadom) [inline rhg_compute_tools/kubernetes.py:113]: Communication

**Comment by delgadom** [comment]
Oh just an import. Ok thanks!

File: rhg_compute_tools/kubernetes.py (hunk 1 above)
Comment (bolliger32) [reply rhg_compute_tools/kubernetes.py]: fixed

**Comment by bolliger32** [comment]
> Oh just an import. Ok thanks!

Yup! L7 in the new version of `kubernetes.py`

**Comment by delgadom** [comment]
I keep hearing about isort all of a sudden. Why is everyone so into sorting imports all of a sudden?

I think if we're going to do any more code formatting, it should be discussed and decided on as an issue, then implemented and enforced with testing, rather than bundled as a one-off change with other fixes. I'm happy going with black and isort across everything but it needs to be a project-level decision and then it needs to be maintained. That sound good? Want to create that issue @bolliger32?

**Comment by bolliger32** [comment]
sure

**Pull request merged**